pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -344,7 +344,7 @@ export function show_modal_dialog(props, footerProps) {
 }
 
 export function apply_modal_dialog(event) {
-    const dialog = event?.target?.closest("[role=dialog]");
+    const dialog = event.target?.closest("[role=dialog]");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18629-20230413-091006-e1ec829d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)